### PR TITLE
fixed return type and added the new meta function

### PR DESCRIPTION
**Pull request by gimbel0893**

### CODE REVIEW

File: server.java [hunk 1]
```diff
@@ -35,8 +35,9 @@
 	/**
 	 * Save the {@link Instrumentation} interface exposed by the JVM.
 	 */
-	public static void premain(String agentArgs, Instrumentation inst) {
+	public static int premain(String agentArgs, Instrumentation inst) {
```
Comment (mjg951) [inline server.java]: Let's use our Integer class instead of the default int

File: server.java [hunk 2]
```diff
@@ -64,4 +65,8 @@ public static Instrumentation getInstrumentation() {
 		return instrumentation;
 	}
 
+	public static Instrumentation getInstrumentationWithMeta() {
+		return instrumentation_meta;
```
Comment (mjg951) [inline server.java]: Let's make this a constant

File: server.java (hunk 1 above)
Comment (gimbel0893) [reply server.java]: Actually I think int is the correct thing to use because.....
Is that alright?

Comment (mjg951) [reply server.java]: Ok good point

**Pull request merged**